docs/redis-clients/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Redis client libraries"
+linkTitle: "Clients"
+description: Connect your application to a Redis database
+weight: 4
+
+---
+
+Here, you will learn how to connect your application to a Redis database. If you're new to Redis, you might first want to understand how to:
+
+* [Get started with Redis](/docs/getting-started/) 
+* [Work with Redis CLI](/docs/manual/cli/)
+* [Batch Redis commands](/docs/manual/pipelining/)
+* [Configure and manage Redis in production](/docs/management/admin/)
+* [Secure your database](/docs/management/security/)
+* [Scale with Redis Cluster](/docs/management/scaling/)
+* [Set up high availability with Redis Sentinel](/docs/management/sentinel/)
+
+For more Redis topics, see [Using](/docs/manual/) and [Managing](/docs/management/) Redis.
+
+If you're ready to get started, tap into the following guides for most popular client libraries you can use with Redis. For a complete list of clients, see [Clients](/resources/clients/).
```

Comment:
We need to figure out how we can implement a visual grid with programming language logos like here https://www.mongodb.com/docs/drivers/

It's much easier to use visual navigation instead of reading the text.

Also, the same language navigation grid should be embedded into the section /docs/getting-started/#using-redis-from-your-application instead of the Ruby example.

Reply:
[](url)